Sources/PointFree/TODO.swift

```diff
@@ -69,3 +72,36 @@ extension CssSelector {
     }
   }
 }
+
+// TODO: Move to HttpPipeline
+public func >¢< <A, B, C, I, J>(
+  lhs: @escaping (A) -> C,
+  rhs: @escaping Middleware<I, J, C, B>
+  )
+  -> Middleware<I, J, A, B> {
+
+    return map(lhs) >>> rhs
```

Comment:
arg so technically this is contramapping down _two_ levels... first level is the function arrow `->` and second level is `Conn` :/ which we could compose these dang things